lib/screens/new_question_screens/new_question_confirmation.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+import 'package:project_eureka_flutter/components/eureka_rounded_button.dart';
+import 'package:project_eureka_flutter/components/side_menu.dart';
+import 'package:project_eureka_flutter/models/question_model.dart';
+
+class NewQuestionConfirmation extends StatefulWidget {
+  final QuestionModel questionModel;
+
+  NewQuestionConfirmation({
+    @required this.questionModel,
+  });
+
+  @override
+  _NewQuestionConfirmationState createState() =>
+      _NewQuestionConfirmationState();
+}
+
+class _NewQuestionConfirmationState extends State<NewQuestionConfirmation> {
+  Container _questionConfirmationBody() {
+    return Container(
+      padding: EdgeInsets.all(20.0),
+      child: Center(
+        child: Column(
+          children: <Widget>[
+            Text(
+              "Question Submitted!",
+              style: TextStyle(fontSize: 24.0),
+            ),
+            SizedBox(
+              height: 20.0,
+            ),
+            _questionConfirmationContainer(),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Text _questionCornfirmationTextStyling(String text) {
+    return Text(
+      text,
+      style: TextStyle(fontSize: 18.0),
+    );
+  }
+
+  Expanded _questionConfirmationContainer() {
+    return Expanded(
+      child: Container(
+        alignment: Alignment.topLeft,
+        width: double.infinity,
+        decoration: BoxDecoration(
+          color: Colors.grey[200],
+          borderRadius: BorderRadius.all(
+            Radius.circular(20.0),
+          ),
+        ),
+        child: Padding(
+          padding: const EdgeInsets.all(20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              _questionCornfirmationTextStyling(widget.questionModel.title),
+              SizedBox(
+                height: 40.0,
+              ),
+              _questionCornfirmationTextStyling(
+                  widget.questionModel.description),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
```

Comment:
Right now, when question is submitted,  we have 1 gray container that shows both title of question and question itself. May i suggest little correction so it's easier to distinguish where title ends :
easy one - make title text bold and maybe one font-size bigger that the question's text
harder one - display question submission in two separate containers (smaller one on top for title, and larger one on the bottom for question itself)